### DISCUSSION TITLE
X32 Record Time Variable no longer working v3.4.3

### DISCUSSION THREAD

**Discussion by sonicnick** [bitfocus/companion#3167] [General]
I had a button text that was set to: 
`$(x32:urec_etime_ms)` X32 record time  and since updating to 3.4.3 it no longer updates or displays anything other than 00:00, same for the VMIX record time: `$(vmix_PC:recording_hms)`

Assume I may have missed something but cant find a box to tick or anything, thanks

**Comment by thedist** [comment]
I can't test for the X32 module, but as for the the vMix `recording_hms` variable that is working perfectly fine from my testing when recording.

Also, issues you may be having should be taken to the respective modules git repo (the `...` button next to a connection has a link to `known issues` which takes you to the correct place), rather than module specific issues in the main Companion repo.

**Reply by sonicnick** [reply]
Ok thanks. Odd.
